docs/guides/setup-dev.md

```diff
@@ -31,13 +40,20 @@ sudo systemctl start docker
 # Foundry
 curl -L https://foundry.paradigm.xyz | bash
 foundryup --branch master
+
+# Clone the repo to the desired location
+git clone git@github.com:matter-labs/zksync-era.git
```

Comment:
There may be ssh dependencies downstream, so it's not very robust.
Instead, I've added instructions to globally override git config to use https instead of ssh.